FAQ.md

```diff
@@ -232,10 +257,14 @@ In case you prefer to use Imgur, then instead of copying the default link that t
 
 ### Q: How to share download links?
 **A**: If you wish to share Google Drive or Mega link then put it inside `[DOWNCLOUD]` tag, Like this : 
-`[DOWNCLOUD]GDRIVE/MEGA LINK[/DOWNCLOUD] 
+`[DOWNCLOUD]GDRIVE/MEGA LINK[/DOWNCLOUD]`
 What it does is shows it in a better way.
 
 If you wish to share a link inside Blackpearl GSuite organization, then you MUST mention about it in the post; otherwise, you'll be warned.
 Posting Gdrive links under any other organization (i.e. edu) are not allowed.
 
 In case you want extra protection, you can use [Blackpearl's Link Protector](https://links.blackpearl.biz/)
+
+### Why this [section](https://blackpearl.biz/forums/116/) have a password?
+This is a forum with tutorial to create profile on various services. This section is locked it behind a password to avoid leechers & copyright trolls. 
+To get the password you need to have 500 likes in your profile, after this plaese go to this [thread](https://blackpearl.biz/threads/907/).
```

Comment:
Grammar-
Line 268: "### Why this [section](https://blackpearl.biz/forums/116/) have a password?" > "### Why does this [section](https://blackpearl.biz/forums/116/) have a password?"
Line 269: "This is a forum with tutorial to create profile on various services. This section is locked it behind a password to avoid leechers & copyright trolls. " > "This is a forum with tutorials to create profiles on various services. This section is locked behind a password to avoid leechers & copyright trolls."
Line 270: "500 likes in your profile" > "500 likes on your profile" 

Spelling-
Line 270: "after this plaese go to this" > "after this please go to this"